var/plugins/ratingmanager.py

```diff
@@ -23,6 +23,21 @@ def getRequiredSpadsVersion(pluginName):
     return requiredSpadsVersion
 
 
+def getBarGameType(teamsize, teamcount):
```

Comment:
I am not familiar enough with this repositor, but is there a way to extract this function somewhere where it could be used by both `ratingmanager.py` and `barmanager.py`?

It would also be nice if the max team size for Small team (5 in this case) could be extracted to some configuration so it can be easily changed, same thing is done on Teiserver already.

Reply:
I'm not sure how to do it. If there's some sample code I could copy I can try.